Netro Sprinklers.indigoPlugin/Contents/Server Plugin/plugin.py

```
@@ -335,6 +341,101 @@ def _get_device_auth(dev):
+                        f"Weather reported to '{dev.name}': "
+                        f"{device_weather.get('t')}{unit_label}, "
+                        f"condition={device_weather['condition']}"
+                    )
+                else:
+                    self.logger.error(
+                        f"Error reporting weather to '{dev.name}': {response}"
+                    )
+
+            except ThrottleDelayError:
+                self.logger.debug(f"Skipping weather report for '{dev.name}' - throttled")
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)